### ISSUE TITLE
align buttons and maintain responsive in mp also

### ISSUE THREAD

**Issue by skanarul8002** [skanarul8002/mca-hub#29]
![Screenshot 2024-10-15 155522](https://github.com/user-attachments/assets/6cad8f8a-b94b-4167-9760-dda82bb605cc)

**Comment by Rahul93102** [comment]
hey, please assign this issue to me.

**Comment by skanarul8002** [comment]
Please @parth-1372, you can check other design-related work or functionalities, like displaying the list of teachers in the admin section as shown in the picture above, and send me a new issue. This issue is already assigned

**Comment by skanarul8002** [comment]
@Rahul93102 and @parth-1372 please read the readme file carefully and work on open issues. definately i will merge your pr. Thank you.

**Comment by Rahul93102** [comment]
<img width="1412" alt="Screenshot 2024-10-19 at 11 32 38 AM" src="https://github.com/user-attachments/assets/7d631454-3152-480f-84e1-3e2d81afdce7">
can you please check this @skanarul8002  ?

**Comment by Rahul93102** [comment]
<img width="1429" alt="Screenshot 2024-10-19 at 4 48 09 @skanarul8002 M" src="https://github.com/user-attachments/assets/66b449c9-0417-46a1-b11c-3dfc4283d709">
@skanarul8002  do i have to add any more changes ,please confirm ?

**Comment by skanarul8002** [comment]
Yes @Rahul93102 ..Please work on all buttons in every section. And  make  them responsive with good alignment

**Comment by Rahul93102** [comment]
@skanarul8002 I think all other sections of buttons is already aligned well

**Comment by skanarul8002** [comment]
Ok @Rahul93102 ...you can work on rest of the buttons
